main/templates/registration/password_change_form.html

```diff
@@ -8,26 +7,40 @@
                 {% csrf_token %}
                 <table>
                     <tr>
-                        <td><label for="id_new_password1">Old password:</label></td>
+                        <td>
+                            <label for="id_new_password1">Old password:</label>
```

Comment:
Not broken in this PR but might as well fix it if it is being updated
```suggestion
                            <label for="id_old_password">Old password:</label>
```